lib/trailblazer/activity/dsl/linear/feature/variable_mapping/dsl.rb

```diff
@@ -188,17 +188,21 @@ class Out < Tuple
               class FiltersBuilder
                 def self.call(user_filter, with_outer_ctx:, **options)
                   if with_outer_ctx
-                    callable    = user_filter # FIXME: :instance_method
-                    call_method = callable.respond_to?(:arity) ? callable : callable.method(:call)
+                    callable = Trailblazer::Option(user_filter) # FIXME: :instance_method
+                    arity = case user_filter
+                            when Symbol then nil
```

Comment:
We could extract that to something `Deprecate.option_arity`, I've needed this before (but will do when needed).

Reply:
Agree, should we do it now or separately ?

Reply:
Later... :laughing: